hyperspy/signals.py

```diff
@@ -33,6 +33,9 @@
     DielectricFunction
         For dielectric function data with signal_dimension equal 1. The signal
         is unbinned by default.
+    HolographyImage
+        For 2D-images taken via electron holography. WaveImages can be
```

Comment:
My bad using copy/paste. I put it together now as it would be only `ComplexSignal2D`, though will change it if we decide to do `ElectronWaveImage`